profiles/TenTraicion.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/103281314/185060662-8b03bbf5-d3dc-4a30-b731-9f5a0424953a.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/185238850-96892427-0bfe-4cdc-ab7f-a87a80720c47.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/185238850-96892427-0bfe-4cdc-ab7f-a87a80720c47.png",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
@Panquesito7, I thought a proper image should be better as it shows the complete design of my GitHub profile. Should I change it to a simpler one like you've provided?

Reply:
> @Panquesito7, I thought a proper image should be better as it shows the complete design of my GitHub profile. Should I change it to a simpler one like you've provided?

I prefer smaller ones, so it is easier to read and understand, but you can use any size you choose. :)


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Inefficient